app/containers/react-native-web/artist/header.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import { FlexAlignType, StyleSheet, Text, TextStyle, View, ViewStyle, Dimensions } from "react-native-web"
+
+// import Events from '../../native_modules/events'
+
+// import colors from "../../../../data/colors"
+const colors = require("../../../../data/colors.json")
+
+import Headline from './text/headline'
+import InvertedButton from './inverted_button'
+import SerifText from './text/serif'
+
+// const isPad = Dimensions.get('window').width > 700
+const isPad = true
+
+interface Props {
+  artist: any
+}
+
+interface State {
+  following: boolean | null,
+  followersCount: number,
+}
+
+class Header extends React.Component<Props, State> {
+  static propTypes: Object = {
+    artist: React.PropTypes.shape({
+      name: React.PropTypes.string,
+      nationality: React.PropTypes.string,
+      birthday: React.PropTypes.string,
+      counts: React.PropTypes.shape({
+        follows: React.PropTypes.number,
+      }),
+    }),
+  };
```

Comment:
Ah yes, in another place I removed them, though, just lost track I guess.

FYI There’s one difference, which is that React’s prop validation happens at runtime.